### PR TITLE
Added GeometryUI/Wpf projects to 2013 solution

### PR DESCRIPTION
**Pull request by Benglin**
These projects were added to only 2012 solution, therefore we don't get them in 2013 solution. This pull request updates 2013 solution. Hi @Randy-Ma, please have a look at it, thanks :)

### CODE REVIEW

File: src/Libraries/DynamoConversions/DynamoConversions.csproj [hunk 1]
```diff
@@ -7,7 +7,7 @@
   <PropertyGroup>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{67CF6CF2-CD6A-442C-BABE-864F896DD8EA}</ProjectGuid>
+    <ProjectGuid>{C5ADC05B-34E8-47BF-8E78-9C7BF96418C2}</ProjectGuid>
```
Comment (Randy-Ma) [inline src/Libraries/DynamoConversions/DynamoConversions.csproj]: @Benglin 
How does it not have any issue before with a wrong guid?

Comment (Randy-Ma) [inline src/Libraries/DynamoConversions/DynamoConversions.csproj]: It turns out that one other project is using the GUID: 67CF6CF2-CD6A-442C-BABE-864F896DD8EA

Comment (Randy-Ma) [inline src/Libraries/DynamoConversions/DynamoConversions.csproj]: And if the guid here is changed, it will be incorrect for the 2012 solution.

**Comment by Benglin** [comment]
Thanks a lot for highlighting this, @Randy-Ma! As you point out, both these solutions must reference to the right project `GUID` value, so I have standardized both to reference `C5ADC05B-34E8-47BF-8E78-9C7BF96418C2`. Both solutions are now loading/building/saving as expected.

**Comment by Randy-Ma** [comment]
@Benglin 
Thanks for the update! LGTM!

**Pull request merged**